client/src/utils/potFlavors.tsx

```diff
@@ -0,0 +1,3 @@
+const POT_FLAVORS = ['Spicy', 'Ox Bone', 'Tomato'];
```

Comment:
Do you think a enum would be better here? You could just reference things by `PotFlavours.OxBone`.
`Object.values(...)` will also let you map them.